### PR TITLE
Get a random picture of RMS working

### PR DESCRIPTION
**Pull request by brb3**
Adds support for a `/stallman` command that returns a random link to Richard Stallman working on his thankpad

### CODE REVIEW

File: Services/StallmanCommand.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            var links = new List<string>();
+            var lines = page.Split("\n");
+            Regex linkRegex;
+            if (image)
+            {
+                linkRegex = new Regex("<a href=\"(.*)\"><img src");
+            }
+            else
+            {
+                linkRegex = new Regex("<a href=\"(pages.*html)\"");
```
Comment (t-richards) [inline Services/StallmanCommand.cs:70]: PARSING HTML WITH REGEX

image

Comment (brb3) [reply Services/StallmanCommand.cs:70]: no u

**Review by brb3 — COMMENTED** [review]
regex is fine, comrade

File: Services/StallmanCommand.cs (hunk 1 above)
Comment (StrangeWill) [reply Services/StallmanCommand.cs:70]: image

Comment (StrangeWill) [reply Services/StallmanCommand.cs:70]: On a serious note, https://html-agility-pack.net/ is usually the go-to when I need to parse HTML, but if regex works then /shrug

**Review by StrangeWill — APPROVED** [review]

**Pull request merged**